tee-worker/omni-executor/rpc-server/src/methods/pumpx/common.rs

```diff
@@ -0,0 +1,98 @@
+use jsonrpsee::types::{ErrorCode, ErrorObjectOwned};
+use parity_scale_codec::Codec;
+use pumpx::types::ApiResponse;
+use serde::Serialize;
+
+use crate::{error_code::*, oneshot, server::RpcContext, Decode};
+use executor_core::native_task::*;
+use native_task_handler::{NativeTaskError, NativeTaskOk, NativeTaskResponse};
+
+#[derive(Serialize, Debug)]
+pub struct PumpxRpcError {
+	pub code: i32,
+	pub message: String,
+}
+
+impl PumpxRpcError {
+	pub fn from_code_and_message(code: i32, message: String) -> Self {
+		Self { code, message }
+	}
+
+	pub fn from_error_code(error_code: ErrorCode) -> Self {
+		let (code, message) = match error_code {
+			ErrorCode::ParseError => (-32700, "Parse error"),
```

Comment:
These messages are all in english while the ones returned from pumpx backend will be i18 , is that correct ?

Reply:
Here is jsonrpsee internal error code. It's fine.